src/main.rs

```diff
@@ -39,9 +41,23 @@ struct Command {
     path: PathBuf,
 
     /// Specifies the Polkadot SDK version. Use '--list' flag to display available versions.
-    #[clap(short, long, required_unless_present = "list")]
+    #[clap(
+        short,
+        long,
+        required_unless_present = "list",
+        required_unless_present = "check"
+    )]
     version: Option<String>,
 
+    /// Checks if deps have the same version as the specified Polkadot SDK version.
+    #[clap(
+        short,
+        long,
+        required_unless_present = "version",
```

Comment:
shouldn't check input a version too? else which version are you checking against?

Reply:
--check does take a version. The `required_unless_present = "version"` was put in place because you don't need to use the --version flag with the --check flag.

psvm --check <VERSION> works.

You don't need to do psvm --check --version <VERSION>

Reply:
ah. i would guess the syntax should be:

```
psvm --version <VERSION> --check
```

So you add check on top of the normal syntax.

Reply:
This is the pattern other tools use, like rustfmt:

https://github.com/rust-lang/rustfmt?tab=readme-ov-file#verifying-code-is-formatted

Reply:
Oh alright got it. Updating the syntax

Reply:
Done @shawntabrizi